Extensions/DialogueTree/JsExtension.js

```diff
@@ -551,19 +615,19 @@ module.exports = {
         'JsPlatform/Extensions/yarn24.png',
         'JsPlatform/Extensions/yarn32.png'
       )
-      .addParameter('string', _('State variable'))
-      .addParameter('string', _('Equal to'))
+      .addParameter('string', _('State variable'), '', false)
+      .addParameter('string', _('Equal to'), '', false)
       .getCodeExtraInformation()
       .setFunctionName('gdjs.dialogueTree.compareVariable');
 
     extension
       .addCondition(
-        'HasClippedTextScrollingCompleted',
-        _('Has clipped text scrolling completed'),
+        'ClippedTextScrollingHasCompleted',
```

Comment:
Same, can we revert to HasClippedTextScrollingCompleted?

Reply:
updated now